README.md

```diff
@@ -61,124 +55,54 @@ VET-CLINIC-DATABASE is microverse project use a relational database to create th
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
-
 - **[PostgreSQL]**
 - **[create animals table]**
 
-
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
-
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
-
-- [Live Demo Link] very soon
+- [Live Demo Link] Very soon
```

Comment:
- [ ] (optional) Since you have not deployed the app and have no live link for the project, it would be prudent to remove the `Live Demo` section from your readme for a professional file